python_modules/dagster/dagster_tests/storage_tests/test_instigation_logger.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from dagster._core.definitions.instigation_logger import InstigationLogger
+
+
+def test_gets_correct_logger(caplog):
+    custom_logger_name = "foo"
+    caplog.set_level(logging.INFO, logger=custom_logger_name)
+
+    instigation_logger = InstigationLogger()
+    assert instigation_logger.name == "dagster"
+
+    instigation_logger = InstigationLogger(logger_name=custom_logger_name)
+    assert instigation_logger.name == custom_logger_name
+
+    # name parameter is passed to metadata
+    instigation_logger = InstigationLogger(name="bar", logger_name=custom_logger_name)
+    assert instigation_logger.name == custom_logger_name
+    assert instigation_logger._name == "bar"  # noqa: SLF001
```

Comment:
this `name` `_name` collision is pretty confusing. @prha  what is the `name` parameter passed to `InstigationLogger` supposed to signify? I see schedules/sensors pass the name of the schedule/sensor as `name`, so is it meant to be the name of the instigator? would it be disruptive to rename it to `instigator_name` or something similar?

Reply:
otherwise i will just leave a comment in the `InstigationLogger` class explaining the difference between the two names, but i want to make sure i explain the the `name` parameter correctly

Reply:
I think it's purely for display purposes in the log output (not the captured logs, the logs emitted in stderr from the grpc server).  I think you can change that to whatever you want.  

The log_key is the important piece that cannot change, which determines the log location.